nilearn/datasets/func.py

```diff
@@ -398,8 +398,8 @@ def fetch_adhd(n_subjects=None, data_dir=None, url=None, resume=True,
     Parameters
     ----------
     n_subjects: int, optional
-        The number of subjects to load. If None is given, all the
-        40 subjects are used.
+        The number of subjects to load from maximum of 40 subjects.
+        If default None, only 30 subjects will be loaded.
```

Comment:
I don't see the rationale of this decision. The original issue was because CircleCI was downloading too much data. Changing the default number of subject will change nothing.